mobly/snippet/client_base.py

```diff
@@ -127,54 +116,42 @@ def start_server(self):
       errors.ServerStartError: when failed to start the snippet server.
     """
 
-    @contextlib.contextmanager
-    def _execute_one_stage(stage):
-      """Context manager for executing one stage.
-
-      Args:
-        stage: StartServerStages, the stage which is running under this
-          context manager.
-
-      Yields:
-        None.
-      """
-      self.log.debug('[START_SERVER] Running the stage %s.', stage.name)
-      yield
-      self.log.debug('[START_SERVER] Finished the stage %s.', stage.name)
-
-    self.log.debug('Starting the server.')
+    self.log.debug('Initializing the snippet package %s.', self.package)
     start_time = time.perf_counter()
 
-    with _execute_one_stage(StartServerStages.BEFORE_STARTING_SERVER):
-      self.before_starting_server()
+    self.log.debug('Preparing starting the snippet server for %s.',
```

Comment:
"Prepare to start ..."

Please make sure all the English sentences in this file are concise and grammatically correct.

Reply:
Sorry, done.